fastf1/events.py

```diff
@@ -746,7 +746,7 @@ def _get_schedule_from_ergast(year) -> "EventSchedule":
                 f"{rnd.get('date', '')}T{rnd.get('time', '')}",
             ).tz_localize(None)
         except dateutil.parser.ParserError:
-            date = pd.NaT
+            date = pd.Timestamp(pd.NaT)
```

Comment:
Do we maybe want to use ``np.datetime64("NaT")`` here? It feels more consistent and seems necessary if we disallow usage of ``pd.NaT``.